src/speaker.js

```diff
@@ -471,6 +475,11 @@ Speaker.prototype = {
   _onAudioTimeUpdateEvent: function (event) {
     var audio = event.currentTarget;
 
+    if (this.startTime) {
+      this.elapsedMilliseconds += ( performance.now() - this.startTime );
```

Comment:
where is `performance` defined?

Reply:
if the user pauses the song and then resumes playback, will this still work?

Reply:
yep - i set the startTime to null when paused

Reply:
`performance` is just js - i think its newer... but it is way faster than `new Date()`

Reply:
The performance.now() method returns a [DOMHighResTimeStamp](https://developer.mozilla.org/en-US/docs/Web/API/DOMHighResTimeStamp), measured in milliseconds.